src/data/move.ts

```diff
@@ -4531,7 +4532,8 @@ export class ForceSwitchOutAttr extends MoveEffectAttr {
 	  	user.scene.triggerPokemonFormChange(switchOutTarget, SpeciesFormChangeActiveTrigger, true);
 
 	  	if (switchOutTarget.hp) {
```

Comment:
I know it's not your code but can we please add readability to this check?

```suggestion
	  	if (switchOutTarget.hp > 0) {
```